src/renderer/scss/component/_header.scss

```diff
@@ -1,64 +1,60 @@
 #header {
-  color: var(--header-color);
-  background: var(--header-bg);
   display: flex;
   align-items: center;
-  justify-content: space-around;
   position: fixed;
-  box-shadow: var(--box-shadow-layer);
   top: 0;
   left: 0;
   width: 100%;
+  height: var(--header-height);
   z-index: 3;
-  padding: $spacing-vertical / 2;
   box-sizing: border-box;
-}
-.header__item {
-  padding-left: $spacing-vertical / 4;
-  padding-right: $spacing-vertical / 4;
-  .button-alt {
-    background: var(--header-button-bg) !important;
-    font-size: 1em;
-  }
-  .button-alt:hover {
-    background: var(--header-button-hover-bg) !important;
-  }
+  color: var(--header-color);
+  background-color: var(--header-bg);
 }
 
-.header__item--wunderbar {
-  flex-grow: 1;
+.header__actions-left {
+  display: flex;
+  padding: 0 5px;
 }
 
-.wunderbar {
-  position: relative;
-  .icon {
-    position: absolute;
-    left: 10px;
-    top: $spacing-vertical / 2 - 4px; //hacked
-  }
+.header__actions-right {
+  margin-left: auto;
 }
 
-.wunderbar--active .icon-search {
-  color: var(--color-primary);
+.header__wunderbar {
+  flex: 1;
+  max-width: 325px;
```

Comment:
Have you tried this larger? Or just using all available space? Already, some results get cut-off, and this seems unnecessary when there is plenty of real estate.

Reply:
Yeah I think we will have it stretch to fill the space